tests/simulator/refined_particle_nbr.py

```
@@ -2,9 +2,7 @@
 #
 # formatted with black
 
-from pyphare.cpp import cpp_lib
-
-cpp = cpp_lib()
+from pyphare import cpp
```

Ruff (0.8.2)

`pyphare.cpp` imported but unused

Remove unused import: `pyphare.cpp`

(F401)